bin/common.ml

```diff
@@ -413,6 +420,12 @@ module Options_implied_by_dash_p = struct
         last
         & opt_all ~vopt:true bool [ false ]
         & info [ "promote-install-files" ] ~docs ~doc)
+    and+ require_dune_project_file =
+      let doc = "Fail if a dune-project file is required but not present." in
```

Comment:
The comment is somewhat recursive. Maybe "Specify if dune-project file is required" or something like this?

Reply:
Alternatively, "Fail if the dune-project file is missing".